docs/nodejs_example.md

```diff
@@ -61,7 +61,9 @@
       });
 
       When('I click on {string}', function (text) {
+        let driver = this.driver;
```

Comment:
Instead of saving this variable, please use an arrow function on the callback passed to `.then` in order to retain scope. Alternatively can use async/await with a requirement of using node 8 for this example.